brush/src/components/InputModal.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import InputROM from './InputROM'
+import { Modal, ModalActions, ModalTitle } from 'former-kit'
+
+const InputModal = () => {
+  const childrenNode = <div>
+    <ModalTitle
+      title="Please, select your ROM file"
+      titleAlign="center"/>
+    <ModalActions children={<InputROM />} />
+  </div>
+
+  return (
+    <Modal
+      children={childrenNode}
```

Comment:
A better approach to pass `children` props is nesting. In this case:

```jsx
<Modal isOpen={true}>
  <div>
    <ModalTitle
      title="Please, select your ROM file"
      titleAlign="center"
    />
    <ModalActions children={<InputROM />} />
  </div>
</Modal>
```

The same approach should be used at `ModalActions`